packages/react-data-grid/index.js

```diff
@@ -1 +1,3 @@
-module.exports = require('./src');
\ No newline at end of file
+import Grid from './src';
+export * from './src';
+export default Grid;
```

Comment:
We need to fix publishing before we publish the next version
https://github.com/adazzle/react-data-grid/blob/next/ci/publish/replacePackageEntry.js#L7